superset/errors.py

```diff
@@ -47,6 +47,7 @@ class SupersetErrorType(str, Enum):
     CONNECTION_HOST_DOWN_ERROR = "CONNECTION_HOST_DOWN_ERROR"
     CONNECTION_ACCESS_DENIED_ERROR = "CONNECTION_ACCESS_DENIED_ERROR"
     CONNECTION_UNKNOWN_DATABASE_ERROR = "CONNECTION_UNKNOWN_DATABASE_ERROR"
+    CONNECTION_DATABASE_PERMISSIONS_ERROR = "TEST_CONNECTION_DATABASE_PERMISSIONS_ERROR"
```

Comment:
You also need to add this to:

- `superset-frontend/src/components/ErrorMessage/types.ts`
- `superset-frontend/src/setup/setupErrorMessages.ts`
- `docs/src/pages/docs/Miscellaneous/issue_codes.mdx`